Topics/Tech_Stacks/Shadcn.md

```diff
@@ -0,0 +1,144 @@
+# Using shadcn/ui
+
+## Table of Contents
+
+#### [Introduction](#introduction-1)
+
+#### [Installation](#installation-1)
+
+#### [Creating and Customizing Components](#creating-and-customizing-components-1)
+
+## Introduction
+
+### What is shadcn/ui? 🤔
+
+[Shadcn/ui](https://ui.shadcn.com/) gives you the ability to use re-usable components that are built using [Radix UI](https://www.radix-ui.com/) and [Tailwind CSS](https://tailwindcss.com/). You are able to choose from their collection of pre-built components, add the code into your project, and then customize them using cva variants.
+
+### What to Expect 📄
+
+This document will go over how to install and integrate shadcn/ui into your project. Then, it will go through an example of building a `Button` component into your project and talk through the details in each step.
+
+## Installation
```

Comment:
List different package managers that are compatible for installing shadcn components (ie., npc, pnpm, yarn, etc.) to reiterate that Step 1 in the example is flexible.